site/src/index.tsx

```diff
@@ -15,7 +16,13 @@ import { Provider } from 'react-redux';
 ReactDOM.createRoot(document.getElementById('root')!).render(
   <React.StrictMode>
     <Provider store={store}>
-      <App />
+      <PostHogProvider
+        apiKey={import.meta.env.VITE_PUBLIC_POSTHOG_KEY}
+        options={{ api_host: import.meta.env.VITE_PUBLIC_POSTHOG_HOST, autocapture: true, enable_heatmaps: true }}
+      >
+        {' '}
```

Comment:
Is this necessary?
```suggestion
```

Reply:
I think I initially codevomited over index.tsx and prettier decided to introduce this "fix" - just removed it and it didn't try to re-add it so will push that change